examples/deployment/README.md

```diff
@@ -200,7 +200,7 @@ OpenShift, through labels and deployment configurations, can support multiple si
 
 4.  Create a service that uses the common label:
 
-        $ oc expose dc/ab-example-a --name=ab-example --selector=ab-example=true
+        $ oc expose dc/ab-example-a --name=ab-example --selector=ab-example=true --generator=service/v1
```

Comment:
I thought we are already defaulting to the service generator if we expose anything other than services.